pii-reporting-ui/src/app/features/confluence-dashboard/confluence-dashboard.component.css

```
@@ -0,0 +1,705 @@
+/* Target p-button host elements inside the group */
+.scan-actions-group > p-button {
+  display: inline-flex;
+}
+
+/* Separator between buttons */
+.scan-actions-group > p-button + p-button {
+  border-left: 1px solid rgba(255, 255, 255, 0.25);
+}
+
+/* Override PrimeNG button styling for all buttons inside the group */
+:host ::ng-deep .scan-actions-group .p-button {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)